### PR TITLE
Update German translation

### PR DESCRIPTION
**Pull request by Niklas04**
Add new phrases and translate them to german, reorder existing ones according to en.json, rename max+ to max_plus

### CODE REVIEW

File: src/translations/de.json [hunk 1]
```diff
@@ -1,24 +1,45 @@
 {
   "status": {
     "cleaning": "Reinigen",
+    "auto": "Automatisches Reinigen",
+    "spot": "Punktreinigung",
+    "edge": "Kantenreinigung",
+    "single_room": "Zimmerreinigung",
     "paused": "Pausiert",
     "idle": "Untätig",
+    "stop": "Stopped",
```
Comment (EdgarKisman) [inline src/translations/de.json:10]: Angehalten

File: src/translations/de.json [hunk 2]
```diff
@@ -28,13 +49,15 @@
...
+    "locate": "Staubsauger lokalisieren",
+    "not_available": "Staubsaugerroboter ist nicht verfügbar"
   },
   "error": {
+    "invalid_config": "Ungeültige Konfiguration",
     "missing_entity": "Angabe der Entität ist erforderlich!"
   },
   "warning": {
-    "actions_array": ""
+    "actions_array": "WARNUNG: 'actions' ist dafür vorgesehen, die Standardaktionen für vorhandene Schaltflächen zu überschreiben. Wenn Sie zusätzliche Aktionen hinzufügen möchten, verwenden Sie stattdessen die Option 'Shortcuts'."
```
Comment (EdgarKisman) [inline src/translations/de.json:60]: `shortcuts` instead of `Shortcuts` as this is a key

**Review by EdgarKisman — CHANGES_REQUESTED** [review]

**Review by denysdovhan — CHANGES_REQUESTED** [review]
Please address the review comments

File: src/translations/de.json (hunk 1 above)
Comment (Niklas04) [reply src/translations/de.json]: Changed - thanks

File: src/translations/de.json (hunk 2 above)
Comment (Niklas04) [reply src/translations/de.json]: Changed - thanks

**Review by EdgarKisman — APPROVED** [review]

**Review by denysdovhan — APPROVED** [review]

**Pull request merged**